examples/angular2/app.ts

```diff
@@ -0,0 +1,86 @@
+/// <reference path="typings/angular2/angular2.d.ts" />
+import {Component, View, bootstrap, For, If} from 'angular2/angular2';
+import {TodoStore, Todo} from 'services/store';
+
+const ESC_KEY = 27;
+const ENTER_KEY = 13;
+
+@Component({
+	selector: 'todo-app',
+	injectables: [
+		TodoStore
+	]
+})
+@View({
+	directives: [For, If],
```

Comment:
This would have to be changed for `NgFor` and `NgIf` (next ng2 release) (and `*ng-for`, `*ng-if`)